app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Runtime Groups
+no_version: true
+---
+
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. You can find a list of all runtime groups in your organization
+on the [Runtime Manager overview](https://konnect.konghq.com/runtime-manager/).
+
+* A **runtime group** is a collection of API connectivity runtime instances
+sharing the same configuration and behavior space.
+
+* A **runtime instance** is a single data plane node with a single instance of
+a runtime, such as Kong Gateway. Runtime instances service traffic for the runtime
+group. All runtime instances in one runtime group
+must be of the same type. Currently, only Kong Gateway runtime types are supported.
+
+Every organization has one default runtime group. With an Enterprise subscription,
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+_[TBA: diagram]_
+
+## Default runtime group
+
+The default runtime group is the foundational group in Konnect. Every
+organization starts with one default group.
+
+This group can be renamed, but it can't be deleted, and its status as the default
+group can't be changed.
+
+### Application registration in the Dev Portal
+
+When publishing documentation from the Service Hub to the Dev Portal, you can
+publish API specs from any version -- no matter which runtime group the version
+is in. However, application registration is only supported for Service versions
+running in the default runtime group.
+
+So, if you want a Service version to be available for application registration,
+create the version in the *default* runtime group. This means that you can have a
+mix of Service versions running in different groups for different purposes.
+
+For example, you might have:
+
+* The latest version of a Service published to the Dev Portal with the latest
+spec, running in the *default* group with application registration enabled.
+
+* A previous version of the Service published to the Dev Portal,
+running in the *default* group, but with application registration disabled.
```

Comment:
Would this allow users that previously registered for this version to continue using their credentials?

Reply:
Uncertain, will test once app reg is fixed. Currently broken and untestable.

Reply:
Update: sort of managed to test. Interesting behavior - if you register an app against a service version and then disable app reg for that version, nothing changes in Dev Portal:

<img width="1385" alt="Screen Shot 2022-03-28 at 2 43 27 PM" src="https://user-images.githubusercontent.com/54370747/160492347-8d4fdd8c-e082-40ba-8e32-44eb3dc0ac3c.png">

All you get is "approved", with no way to know what the status of the service version is. Unfortunately, now Runtime Manager isn't recognizing any runtime instances, so I can't actually test the credential.

Reply:
Another update, finally got through and was able to test. Answer is that disabling app reg does nothing to existing applications. If you have a credential for an older version of the service, you can continue using it.